python/py-ml_dtypes/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ml_dtypes
+version             0.5.3
+revision            0
+categories-append   devel
+license             Apache-2
+
+maintainers         nomaintainer
+
+description         NumPy dtype extensions used in machine learning
+
+long_description    {*}${description}
+
+homepage            https://github.com/jax-ml/ml_dtypes
+
+checksums           rmd160  ee23e4c842979fdb630630fa0eb53b1bb542c43c \
+                    sha256  95ce33057ba4d05df50b1f3cfefab22e351868a843b3b15a46c65836283670c9 \
+                    size    692316
+
+python.versions     38 39 310 311 312 313
+
+build.args-append   --skip-dependency-check
```

Comment:
this is never correct; please resolve the underlying issue

Reply:
Thanks, fixed.